libnmstate/ifaces/ifaces.py

```diff
@@ -325,6 +326,17 @@ def _validate_unknown_parent(self):
                     f"{iface.parent}"
                 )
 
+    def _remove_unknown_interfaces(self):
+        """
+        Remove unknown type interfaces that come from the current state.
+        """
+        for iface in self.current_ifaces.values():
+            if (
+                not iface.is_desired
+                and iface.iface_type == InterfaceType.UNKNOWN
+            ):
+                self._ifaces.pop(iface.name, None)
```

Comment:
A logging before this line could help debuging.

Reply:
Done, thanks!